Package.swift

```diff
@@ -20,14 +20,7 @@ let package = Package(
         .target(
             name: "Valet",
             dependencies: []),
-        .target(
-            name: "LegacyValet",
-            dependencies: [],
-            publicHeadersPath: "Public"),
-        .testTarget(
-            name: "ValetTests",
```

Comment:
Unfortunately, having the `ValetTests` target in the Package caused the `watchOS` build to fail, since `XCTest` does not exist in `watchOS`. The tests are still run via the manually managed `Valet.xcodeproj`, so we don't lose much by removing this test target here.

Reply:
Bummer

Reply:
Agreed. Excited to get these tests back